src/components/requirements/DataBlock.tsx

```diff
@@ -0,0 +1,42 @@
+import { Skeleton } from "@/components/ui/Skeleton";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from "@/components/ui/Tooltip";
+import { Warning } from "@phosphor-icons/react/dist/ssr";
+import type { PropsWithChildren } from "react";
+
+type Props = {
+  isLoading?: boolean;
+  error?: string;
+};
+
+const DataBlock = ({
+  isLoading,
+  error,
+  children,
+}: PropsWithChildren<Props>): JSX.Element => {
+  if (isLoading) return <Skeleton className="inline w-24" />;
+
+  return (
+    <Tooltip open={error ? undefined : false}>
```

Comment:
Actually, I'm not really sure if we need the tooltip & error state here, since we'll probably use it for static data only. What do you think?